version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexual language or imagery
+* Derogatory comments
+* Personal attacks
+* Trolling or insulting comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without permission
+* Other unethical or unprofessional conduct.
```

Comment:
Two more loopholes that came to mind:

1. Maintainers sympathetic with the harasser can argue that it's not really harassment and that's not how _they_ interpret the Code of Conduct.
2. Contributors/maintainers who have a grudge against another contributor/maintainer waiting for them to slip up, and arguing that their minor transgression violates the Code of Conduct and should be banned.

I would like to see these loopholes closed. I disagree that enumerating behavior or clarifying language is futile, as that is what we've been doing in the issues so far and it's yielded results.